debian/rules.in

```diff
@@ -31,40 +33,41 @@ export TIME=$(shell LANG=C date --date='@$(TIMESTAMP)' '+%T')
 kernel_version = @KERNEL_VERSION@
 configure_realtime_arg = @CONFIGURE_REALTIME_ARG@
 enable_build_documentation = @ENABLE_BUILD_DOCUMENTATION@
+SRCDIR = $(CURDIR)/src
 DESTDIR=$(CURDIR)/debian/tmp
 DEV_PACKAGE_NAME=@MAIN_PACKAGE_NAME@-dev
 
+export PYTHON=/usr/bin/python3
+
 %:
-	dh $@
+	dh $@ -D$(SRCDIR)
 
 override_dh_auto_configure:
 	cd src && ./autogen.sh
-	cd src && PYTHON=/usr/bin/python3 ./configure \
+	dh_auto_configure -- \
 	    --prefix=/usr --sysconfdir=/etc \
+	    --libdir=/usr/lib \
+	    --disable-multiarch \
```

Comment:
I believe we should avoid --disable-multiarch.  Any idea how to do that?

Reply:
In fact, this is the application of configuration parameters explicitly.
I just brought the problem to the surface.

We can go back to the beginning and write like this:
```make
override_dh_auto_configure:
 	cd src && ./autogen.sh
	cd src && PYTHON=/usr/bin/python3 ./configure \
```
or so:
```make
override_dh_auto_configure:
 	cd src && ./autogen.sh
	dh_configure -D$(SRCDIR) -- \
```
And these configuration parameters will be applied implicitly.

I have not found any description or reference on this issue in the linuxcnc documentation.
There is also no agreement or description about where which files should be located.
So just reading [this: #multiarch](https://www.debian.org/doc/manuals/maint-guide/advanced.en.html#multiarch) and [this: MultiarchSpec](https://wiki.ubuntu.com/MultiarchSpec)

Reply:
I guess my problem is that I do not understand how disabling multiarch improve parallelism in the build.  can you explain?

Reply:
This does not affect the parallel build in any way.
This affects the parallel execution of the configuration script, but has a small effect.

I will formulate your question in a different way.
What should we additionally do to add support for multiarchitectural assembly?

When we write:
```make
override_dh_auto_configure:
 	cd src && ./autogen.sh
	cd src && PYTHON=/usr/bin/python3 ./configure \
```
This means that the script needs to do literally what is written below and not apply any implicit rules, including parallel assembly and multiarchitecture support.
When we write:
```make
override_dh_auto_configure:
 	cd src && ./autogen.sh
 	dh_auto_configure -- \
 	    --prefix=/usr --sysconfdir=/etc \
	    --libdir=/usr/lib \
	    --disable-multiarch \
```
This means dh_auto_XX to apply all implicit rules except those that are explicitly specified as not to apply.

In other words, before this PR there was no multiarchitectural support
and I just turned it off explicitly in order for the project to be assembled.

Reply:
Going back to that would be a compromise.
```make
override_dh_auto_configure:
 	cd src && ./autogen.sh
	cd src &&  ./configure \
```